jest.config.js

```diff
@@ -156,8 +156,8 @@ module.exports = {
 
   // The regexp pattern or array of patterns that Jest uses to detect test files
   testRegex: [
-  	/\.test\.js?$/
-	],
+    /test\/.*\.test\.js$/
+  ],
```

Comment:
I think these changes are not really needed. I was able to run Jest with these changes removed.

Reply:
sometimes stryker leaves a `.stryker-tmp/...` directory after running it (reproducible when hitting `Ctrl+C`), running jest after that will also run the tests in those directories.

Also it just speeds up jest since it doesn't need to search for tests anywhere else.